app/src/main/java/com/example/sharingang/items/ItemsViewModel.kt

```diff
@@ -21,6 +21,14 @@ class ItemsViewModel @Inject constructor(
     val navigateToEditItem: LiveData<Item?>
         get() = _navigateToEditItem
 
+    private val _focusedItem = MutableLiveData<Item?>()
```

Comment:
Just out of curiosity, what does it mean to have an "_" before some variable names? For mutable variables?

Reply:
I think it's just a notation for private variable.